ui/src/app/cohort-search/search-group/search-group.component.ts

```diff
@@ -39,10 +39,12 @@ export class SearchGroupComponent {
     this.actions.removeGroup(this.role, this.groupId);
   }
 
-  launchWizard(criteriaType: string) {
+  launchWizard(criteria: any) {
```

Comment:
Would it make sense to use a type here and make constants.ts use an object with constructor?

Reply:
I agree, we should do this in the future. I'll create a ticket for it.